Lab07.py

```diff
@@ -0,0 +1,17 @@
+import urllib.request
+
+class File:
+  def __init__ (self, url):
+    self.data = urllib.request.urlopen(url).read().decode("utf-8")

```

Comment:
if the file is 10GB?

Reply:
I cant find so large text file. Could you send me a link?

Reply:
you don't need 10GB file here to understand what will happen. On the other hand if you need it, you can generate one